### PR TITLE
result

### PR DESCRIPTION
**Pull request by Andrew77788**
- [DEMO LINK](https://Andrew77788.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://Andrew77788.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/card.scss [hunk 1]
```diff
@@ -0,0 +1,102 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  font-family: Roboto, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/styles/card.scss:8]: apply `font-family` for html tag selector 
```suggestion
```

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import '../styles/utils/variables';
+@import '../styles/card';
+@import '../styles/stars';
+
+
+
+
```
Comment (SanyaBratashchuk) [inline src/styles/index.scss:7]:
```suggestion
```

File: src/index.html [hunk 3]
```diff
@@ -6,8 +6,46 @@
...
+      <img src="../src/images/imac.jpeg" alt="imac">
+    </div>
+
+    <div class="card_body">
+      <div class="card_body_title">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card_body_product-code">
+        Product code: 195434
+      </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:25]: use semantic tags like `h3`,  `p` for text

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -6,8 +6,46 @@
...
+          Reviews: 5
+        </div>
+      </div>
+      <div class="card_body_price">
+        <div class="card_body_price_title">
+          Price:
+        </div>
+        <div class="card_body_price_price">
+          $2,199
+        </div>
```
Comment (etojeDenys) [inline src/index.html:44]: better to use some semantic tags for the text

**Review by etojeDenys — APPROVED** [review]
good job

**Pull request closed without merging**